gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Main properties
 group=com.truelayer
 archivesBaseName=truelayer-java
-version=13.3.0
+version=14.0.0
```

Comment:
Moved some classes around (e.g. created packages `userselected` and `preselected` inside `schemeselection` package)